src/components/dropdown/dropdown.tsx

```diff
@@ -90,9 +90,7 @@ function Dropdown(props: DropdownProps) {
                     data-testid={`dropdown-knapp_${id}`}
                     aria-label={ariaLabel()}
                 >
-                    <BodyShort size="small" className="dropdown__btntext">
-                        {name}
-                    </BodyShort>
+                    <BodyShort size="small">{name}</BodyShort>
                 </button>
```

Comment:
Stiller bare spm for å lære litt, håper det er ok. Var klassenavnet brukt til å lage noen stil på knappen før eller var det bare lagt til uten at det egentlig ble brukt til noe? className brukes bare til styling?

Reply:
Egentlig samme spørsmål for de andre klassene "nav-input" og "arbeidsliste-modal__tittel"

Reply:
Hadde helt glemt å sjekke om det hadde skjedd noe i disse PRene 🤦‍♂️ 
Det er ikke helt godt å si egentlig, i dette tilfellet så tror jeg det kan ha eksistert noe CSS tidligere som har blitt fjerna, men i andre tilfeller så tror jeg klasser bare er lagt på for å ha dem tilgjengelig 🤷

Reply:
nav-input tas vekk fordi den ikke har noen effekt lenger, den klassen har nok noe med det gamle designsystemet å gjøre